api/src/main/java/org/openmrs/module/fhir/api/RelatedPersonService.java

```diff
@@ -28,6 +30,14 @@
 	 */
 	RelatedPerson getRelatedPerson(String uuid);
 
+	/**
+	 * Get related person by id
+	 *
+	 * @param identifier The identifier of patient
+	 * @return a List RelatedPerson fhir resource
+	 */
```

Comment:
Can you add a `@since` annotation for when this method was introduced? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

Reply:
The value of the since annotation should be the version of the module when the method was introduced. That is the version of the module that we are going to release.

Reply:
Did you see the above comment?

Reply:
Same here